src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,687 @@
+				"HH": hours field in 24-hours format with leading zero
+				"mm": minutes field with leading zero
+				Note! This option can not be set runtime.
+				```
+					//Initialize
+					$(".selector").%%WidgetName%%({
+						timeInputFormat : "time"
+					});
+
+					//Get
+					var timeInputFormat = $(".selector").%%WidgetName%%("option", "timeInputFormat");
+				```
+			*/
+			timeInputFormat: null,
+			/* type="string"
+				Gets/Sets format of time while timepicker has no focus.
+				Value of that option can be set to a specific time pattern or to a flag defined by regional settings.
+				If value is not set, then the timeInputFormat is used automatically.
+				If value is set to explicit time pattern and pattern besides time-flags has explicit characters which match with time-flags or mask-flags, then the "escape" character should be used in front of them.
+				List of predefined regional flags:
+				"time": the timePattern member of regional option is used
+				List of explicit characters, which should have escape \\ character in front of them:
+					C, &, a, A, ?, L, 9, 0, #, >, <, y, M, d, h, H, m, s, t, f.
+				List of time-flags when explicit time pattern is used:
+				"t": first character of string which represents AM/PM field
+				"tt": 2 characters of string which represents AM/PM field
+				"h": hours field in 12-hours format without leading zero
+				"hh": hours field in 12-hours format with leading zero
+				"H": hours field in 24-hours format without leading zero
+				"HH": hours field in 24-hours format with leading zero
+				"m": minutes field without leading zero
+				"mm": minutes field with leading zero
+				```
+					//Initialize
+					$(".selector").%%WidgetName%%({
+						timeDisplayFormat: "time"
+					});
+
+					//Get
+					var timeDisplayFormat = $(".selector").%%WidgetName%%("option", "timeDisplayFormat");
+
+					//Set
+					$(".selector").%%WidgetName%%("option", "timeDisplayFormat", "time");
+				```
+			*/
+			timeDisplayFormat: null,
+			/* type="bool" Gets/Sets if the editor should only allow values from the list of items. Matching is case-insensitive.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					isLimitedToListValues : true
+				});
+
+				//Get
+				var limited = $(".selector").%%WidgetName%%("option", "isLimitedToListValues");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "isLimitedToListValues", false);
+			```*/
+			isLimitedToListValues: true,
+			/* type="bool" Disables/Enables default notifications for basic validation scenarios built in the editors such as required list selection, value wrapping around or spin limits.
+			```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					suppressNotifications : true
+				});
+
+				//Get
+				var suppressNotifications = $(".selector").%%WidgetName%%("option", "suppressNotifications");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "suppressNotifications", true);
+			```
+			*/
+			suppressNotifications: false,

```

Comment:
Why do you have those options defined here, when they come from the base widget? Or there is a specific reason which I'm missing?